docs/plugins/graphback-client.md

```diff
@@ -59,7 +58,7 @@ Example plugin configuration in YAML:
  plugins:
   graphback-client:
     format: 'graphql'
-    outputPath: ./client/src/graphql
+    outputPath: ./client/src/graphql/graphback.graphql
```

Comment:
generated.graphql?

Reply:
Does it matter? The name can be changed and `graphback.graphql` doesn't seem bad